moveit_ros/visualization/motion_planning_rviz_plugin/src/ui/motion_planning_rviz_plugin_frame.ui

```diff
@@ -795,10 +797,12 @@
                </size>
               </property>
               <property name="toolTip">
-               <string>If checked, a new IK solution is calculated whenever the robot state is in collision. This setting only affects the state set via the interactive marker. Planning is always aware of collisions.</string>
+               <string>If checked, the IK solver tried to avoid collisions when searching for the state set via the interactive marker.
+
+This is usually achieved by random seeing, which can flip the robot configuration (e.g. elbow up/down). Note that motion planning is always collision-aware, regardless of this checkbox.</string>
```

Comment:
```suggestion
               <string>If checked, the IK solver tries to avoid collisions when searching for a start/end state set via the interactive marker.

This is usually achieved by random seeding, which can flip the robot configuration (e.g. elbow up/down). Note that motion planning is always collision-aware, regardless of this checkbox.</string>
```